code/modules/mob/living/carbon/alien/alien_defenses.dm

```diff
@@ -1,5 +1,5 @@
-/mob/living/carbon/alien/hitby(atom/movable/AM)
-	..(AM, 1)
+/mob/living/carbon/alien/hitby(atom/movable/AM, skipcatch, hitpush)
+	..(AM, hitpush = 0)
```

Comment:
Intentionally left this out. I want aliens to be able to catch things, it's the most hilarious "bug" i know of off the top of my head, maybe second to the morgue trays floating around. It's so incredibly rare that it happens that it always makes someone's day.